api/server/sdk/cloud_backup.go

```diff
@@ -133,7 +134,11 @@ func (s *CloudBackupServer) GroupCreate(
 
 		// Check ownership for this intersection
 		if err := checkAccessFromDriverForVolumeIds(ctx, s.driver(ctx), volumesToCheck, api.Ownership_Read); err != nil {
-			return nil, err
+			if err == kvdb.ErrNotFound {
```

Comment:
checkAccessFromDriverForVolume returns "codes.PermissionDenied" for access errors.  Perhaps we should check for that and return other errors as is?

Reply:
checkAccessFromDriverForVolumeIds is used by other functions, so I don't want to return the error about GroupIDs there.

But yeah I'll change the caller to check for codes.NotFound instead.